manifests/plugin/remote_execution.pp

```diff
@@ -1,9 +1,21 @@
 # Installs foreman_remote_execution plugin
-class foreman::plugin::remote_execution {
+#
+# === Parameters:
```

Comment:
I think this can be ```Advanced``` so ```foreman-installer --help``` doesn't show it. It's rather internal and I'd expect no foreman-installer user to ever need this.